lib/xeroizer/partner_application.rb

```diff
@@ -42,5 +42,14 @@ def initialize(consumer_key, consumer_secret, path_to_private_key, path_to_ssl_c
         end
       end
     
+    private
+
+      def read_certificate(cert)
+        if cert[-4..-1] == '.pem'
```

Comment:
Perhaps didn't feel quite confident passing possible certificate string into the File.exists? :)
i.e. it attempts to check if cert is the path or an actual, preloaded, certificate.